module/dice/_module.mjs

```diff
@@ -1,3 +1,4 @@
 export {default as D20Roll} from "./d20-roll.mjs";
 export {default as DamageRoll} from "./damage-roll.mjs";
 export * from "./dice.mjs";
+export {default as simplifyRollFormula} from "./simplify-roll-formula.mjs";
```

Comment:
I don't really understand the reason to move this, but it's not a problem necessarily. It felt to me like simplification of a roll formula belonged in the `dice` module with other dice-related helpers.